### ISSUE TITLE
Vue 3 Support

### ISSUE THREAD

**Issue by rhclayto** [technowledgy/vue-h5p#156]
Will a Vue 3 version be forthcoming, by any chance?

**Comment by rhclayto** [comment]
Thanks for the response. I will try it out in the next few days, & report what I discover.

**Comment by Mhyrtas** [comment]
I've been trying vue-h5p on Vue 3 and ran into a problem. Seems like it can't generate the iframe cause "srcdoc" is undefined. 

The only problem I can see is that my src prop is not right?

**Comment by vsojo** [comment]
Same problem as @Mhyrtas here :(
`runtime-core.esm-bundler.js:496 TypeError: Cannot read properties of undefined (reading '_c')`
The `undefined` is `srcdoc`

**Comment by NLegendre** [comment]
Hello, I tried to install the beta version for Vue3, but I have this error in the console : 

`vue-h5p.js?v=baf16da7:8554 Uncaught (in promise) TypeError: Cannot read properties of null (reading 'ce')`

Do you have a Vue3 example? Maybe I missed something in the configuration?

Thank you for your help

**Comment by NLegendre** [comment]
Yes of course : https://stackblitz.com/edit/vitejs-vite-erfch7zq?file=src%2FApp.vue

This is the given example. If you open the console, you can see the error.

**Comment by NLegendre** [comment]
Hello,
Thanks for the update.
I've changed the example on Stackblitz with the beta.2 (https://stackblitz.com/edit/vitejs-vite-erfch7zq?file=src%2FApp.vue)
The error still occurs (I tried to run the example on Chrome and Firefox).

FYI: I’m currently using the h5p-standalone JS library, so there’s no urgency to fix this issue if you don’t have time.

**Comment by wolfgangwalther** [comment]
This is now fixed in `beta.3`. This new stackblitz https://stackblitz.com/edit/vitejs-vite-iqa8gjyt works for Firefox. It doesn't work for Chrome for me, but when I download it and run it locally it works there, too. So this seems to be some stackblitz/Chrome related problem to me.
